src/Symfony/Component/Security/Core/Tests/Authentication/Provider/RememberMeAuthenticationProviderTest.php

```diff
@@ -26,12 +26,15 @@ public function testSupports()
         $this->assertFalse($provider->supports($this->getMockBuilder('Symfony\Component\Security\Core\Authentication\Token\TokenInterface')->getMock()));
     }
 
+    /**
+     * @expectedException \Symfony\Component\Security\Core\Exception\BadCredentialsException
```

Comment:
you should also use `@expectedExceptionMessage` to be sure it is the right one being thrown

Reply:
Thanks, added.